ledger/participant-integration-api/src/main/resources/db/migration/postgres/V39__Participant-pruning.sql

```diff
@@ -0,0 +1,12 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V39: Pruned offset tracking
+--
+-- This schema version adds tracking of the pruned portion of the participant ledger to the
+-- parameters table.
+---------------------------------------------------------------------------------------------------
+
+-- Add the column for most recent pruning offset to parameters.
+ALTER TABLE parameters ADD COLUMN participant_pruned_up_to_inclusive BYTEA;
```

Comment:
If `NULL` means "no prune happened so far" this should be fine. Is that so?

Reply:
Correct